docs/source/ContributionGuide.rst

```diff
@@ -65,10 +65,134 @@ be submitted as a github Pull Request. This process tries to ensure that every
 contributed patch goes through the CI jobs before it is considered good for
 inclusion.
 
+Git workflow
+------------
+
+- Fork the repository in github.
+
+- Clone from your fork::
+
+    $ git clone git@github.com:<username>/avocado.git
+
+- Enter the directory::
+
+    $ cd avocado
+
+- Create a ``remote``, pointing to the upstream::
+
+    $ git remote add upstream git@github.com:avocado-framework/avocado-misc-tests.git
```

Comment:
I guess you intended to point to `git@github.com:avocado-framework/avocado.git`, right?